full-stack-tests/core/src/frontend/standalone/Error.test.ts

```diff
@@ -0,0 +1,50 @@
+import { LoggingMetaData, ProcessDetector } from "@itwin/core-bentley";
+import { TestUtility } from "../TestUtility";
+import { TestUsers } from "@itwin/oidc-signin-tool/lib/cjs/TestUsers";
+import { coreFullStackTestIpc } from "../Editing";
+import { InUseLock, ITwinError, LockState } from "@itwin/core-common";
+import { expect } from "chai";
+
+if (ProcessDetector.isElectronAppFrontend) {
+
+  describe("Error (#integration)", async () => {
+
+    beforeEach(async () => {
+      await TestUtility.startFrontend();
+      await TestUtility.initialize(TestUsers.regular);
+    });
+
+    afterEach(async () => {
+      await TestUtility.shutdownFrontend();
+    });
+
+    it("should receive ConflictingLocksError on the frontend", async () => {
```

Comment:
AFAICT `ConflictingLocksError` is not involved at all.

Reply:
You're right, changed to InUseLocksError